RecoBTag/Skimming/interface/BTagSkimMC.h

```diff
@@ -1,7 +1,7 @@
-#include "FWCore/Framework/interface/EDFilter.h"
+#include "FWCore/Framework/interface/one/EDFilter.h"
 #include "FWCore/ParameterSet/interface/ParameterSet.h"
 
-class BTagSkimMC : public edm::EDFilter {
+class BTagSkimMC : public edm::one::EDFilter<> {
```

Comment:
Why one? From a quick glance this could be a stream or maybe a global, both of which are much more efficient than one.

Reply:
Hi @bsunanda , are you planning to update the PR with this suggestion? If so, I'll wait before signing. Thanks

Reply:
Please wait. I shall update the PR with suggestion from Chris